neqo-udp/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "neqo-udp"
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+
+[lints]
+workspace = true
+
+[dependencies]
+log = { workspace = true }
+neqo-common = { path = "./../neqo-common" }
+quinn-udp = { version = "0.5.0", default-features = false }
+tokio = { version = "1", default-features = false, features = ["net"], optional = true }
+
+[dev-dependencies]
+tokio = { version = "1", default-features = false, features = ["net", "time", "macros", "rt", "rt-multi-thread"] }
```

Comment:
Why is `tokio` also a `dev-dependency`?

Reply:
Presumably because the dev variant has more features enabled.

The lack of an EOL here is also worth fixing.  You can configure your editor to add these to all text files.

Reply:
Correct.

Firefox does not need `tokio`. `neqo-client` and `neqo-server` in `neqo-bin` do need `tokio`, but only the `net` `feature`. Thus `tokio` is an `optional` `dependency` restricted to the `net` `feature` only. 

``` toml
[dependencies]
# [...]
tokio = { version = "1", default-features = false, features = ["net"], optional = true }
```

In addition, the `neqo-udp` tests always need `tokio` and they make use of various features beyond `net`, thus the additional import as a `dev-dependency` with more features enabled.

```
[dev-dependencies]
tokio = { version = "1", default-features = false, features = ["net", "time", "macros", "rt", "rt-multi-thread"] }
```

> The lack of an EOL here is also worth fixing. You can configure your editor to add these to all text files.

Will do!